tests/tx_test.py

```diff
@@ -1148,3 +1148,13 @@ def test_cidkeyed_lib_missing(file):
     ufo_input_path = get_input_path(folder + file)
     arg = [TOOL, '-t1', '-f', ufo_input_path]
     assert subprocess.call(arg) == 6
+
+
+def test_cff2_windows_line_endings_bug1355():
+    # Testing writing binary to stdout on Windows
+    # to ensure line endings are not inserted.
+    font_path = get_input_path('regular_CFF2.otf')
+    actual_path = runner(CMD + ['-s', '-a', '-o', 'cff2',
+                                '*S', '*b', '-f', font_path])
+    expected_path = get_expected_path('bug1355.CFF2')
```

Comment:
I suggest renaming the file to `bug1355.cff2`, i.e. with a lowercase `.cff2` for consistency with the other `*.cff2` files in [expected_output](https://github.com/adobe-type-tools/afdko/tree/develop/tests/tx_data/expected_output) and with the [.gitattributes](https://github.com/adobe-type-tools/afdko/blob/develop/.gitattributes) file.